components/src/core/components/Input/Select/SelectInputButton.vue

```diff
@@ -50,6 +50,21 @@
               />
               <span>{{ $vt(buttonData.labelMini) }}</span>
             </div>
+            <div
+              class="d-flex
```

Comment:
what happens if these props are not passed? probably will need to render this conditionally if `additionalIconName` prop is set

Reply:
rendered conditionally based on passing the additional icon